pilot/cmd/pilot-discovery/main.go

```diff
@@ -150,6 +142,17 @@ func init() {
 	discoveryCmd.PersistentFlags().IntVar(&monitoringPort, "monitoringPort", 9093,
 		"HTTP port to use for the exposing pilot self-monitoring information")
 	discoveryCmd.PersistentFlags().MarkDeprecated("monitoringPort", "Use --monitoringAddr instead")
+	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableProfiling, "profile", true,
+		"Enable profiling via web interface host:port/debug/pprof")
+	discoveryCmd.PersistentFlags().MarkDeprecated("profile", "")
+	discoveryCmd.PersistentFlags().BoolVar(&serverArgs.DiscoveryOptions.EnableCaching, "discovery_cache", true,
+		"Enable caching discovery service responses")
+	discoveryCmd.PersistentFlags().MarkDeprecated("discovery_cache", "")
+	// TODO (rshriram): Need v1/v2 endpoints and option to selectively
+	// enable webhook for specific xDS config (cds/lds/etc).
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.WebhookEndpoint, "webhookEndpoint", "",
```

Comment:
@rshriram: please confirm that this a v1 only flag.

Reply:
it isn't used anywhere.

Reply:
Its v1 only.. and I sugest removing this..